EventFilter/L1TRawToDigi/plugins/implementations_stage2/ZDCUnpacker.h

```diff
@@ -0,0 +1,23 @@
+#ifndef L1T_PACKER_STAGE2_ZDCUnpacker_H
+#define L1T_PACKER_STAGE2_ZDCUnpacker_H
+
+#include "EventFilter/L1TRawToDigi/interface/Unpacker.h"
+
+namespace l1t {
+  namespace stage2 {
+    class ZDCUnpacker : public Unpacker {
+    public:
+      ZDCUnpacker();
+      ~ZDCUnpacker() override{};
+
+      bool unpack(const Block& block, UnpackerCollections* coll) override;
+
+      virtual inline void setEtSumZDCCopy(const unsigned int copy) { EtSumZDCCopy_ = copy; };

```

Comment:
why `virtual` ?

Reply:
It's virtual since the similar function in etsumunpacker was virtual.

Reply:
But `setEtSumZDCCopy()` is implemented only in a single class - there is no inheritance involved.
Please remove the `virtual` function specifier here.
A separate PR can be made to fix the existing code.